src/__init__.py

```diff
@@ -121,26 +129,55 @@ def fputs(string):
             putchar(' ')
             col += 1
         while col <= colright:
-            if numlines == 6:
-                char = background6[lin][col]
-            elif numlines == 18:
-                char = background18[lin][col]
-            elif numlines == 19:
-                char = background19[lin][col]
-            elif numlines == 21:
-                char = background21[lin][col]
-            elif numlines == 22:
-                char = background22[lin][col]
-            elif numlines == 23:
-                char = background23[lin][col]
-            elif numlines == 24:
-                char = background24[lin][col]
-            elif numlines == 29:
-                char = background29[lin][col]
-            elif numlines == 32:
-                char = background32[lin][col]
-            else:
-                char = '@'
+            if hemisphere == 'north':
+                # north - read moons from upper-left to bottom-right
+                if numlines == 6:
+                    char = background6[lin][col]
+                elif numlines == 18:
+                    char = background18[lin][col]
+                elif numlines == 19:
+                    char = background19[lin][col]
+                elif numlines == 21:
+                    char = background21[lin][col]
+                elif numlines == 22:
+                    char = background22[lin][col]
+                elif numlines == 23:
+                    char = background23[lin][col]
+                elif numlines == 24:
+                    char = background24[lin][col]
+                elif numlines == 29:
+                    char = background29[lin][col]
+                elif numlines == 32:
+                    char = background32[lin][col]
+                else:
+                    char = '@'
+            else: 
+                # south - read moons from bottom-right to upper-left
+                # equivalent to rotate 180 degress or turn upside-down
+                if numlines == 6:
+                    char = background6[-1-lin][-col]
+                elif numlines == 18:
+                    char = background18[-1-lin][-col]
+                elif numlines == 19:
+                    char = background19[-1-lin][-col]
+                elif numlines == 21:
+                    char = background21[-1-lin][-col]
+                elif numlines == 22:
+                    char = background22[-1-lin][-col]
+                elif numlines == 23:
+                    char = background23[-1-lin][-col]
+                elif numlines == 24:
+                    char = background24[-1-lin][-col]
+                elif numlines == 29:
+                    char = background29[-1-lin][-col]
+                elif numlines == 32:
+                    char = background32[-1-lin][-col]
+                else:
+                    char = '@'
```

Comment:
Actually, my original idea was to stay to the 1973 `cphoon` of Jeff as close as possible,
but if we change the code anyway we can write this code much more concise (in Python).
Or maybe you are right, and we should leave the spirit of the code as is